drivers/spi/Kconfig.andes_atcspi200

```diff
@@ -11,3 +11,12 @@ config SPI_ANDES_ATCSPI200
 	depends on DT_HAS_ANDESTECH_ATCSPI200_ENABLED
 	help
 	  Enable driver for Andes ATCSPI200 SPI controller
+
```

Comment:
change your commit title to: "drivers: spi: Support dma mode for atcspi200"

Reply:
OK, I got it.
Thank for your recommendation, we had done the modification.

Reply:
I still read "drivers: spi: Support dma mode"